frontend/src/components/ProjectList/components/Dataset/components/MoreInformation/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react";
+import { Project } from "src/common/entities";
+import { StyledAnchor, Wrapper } from "./style";
+
+interface Props {
+  links: Project["links"];
+}
+
+const MoreInformation: FC<Props> = ({ links }) => {
+  return (
+    <Wrapper>
+      {links.map(link => {
+        return (
+          <StyledAnchor
+            key={link.url}
+            href={link.url}
+            target="_blank"
```

Comment:
why _blank?

Reply:
to open the link in a new tab instead of the current tab 😄